src/main/kotlin/no/skatteetaten/aurora/gobo/integration/naghub/NagHubService.kt

```diff
@@ -0,0 +1,64 @@
+package no.skatteetaten.aurora.gobo.integration.naghub
+
+import com.fasterxml.jackson.annotation.JsonValue
+import com.fasterxml.jackson.databind.JsonNode
+import mu.KotlinLogging
+import no.skatteetaten.aurora.gobo.ServiceTypes
+import no.skatteetaten.aurora.gobo.TargetService
+import no.skatteetaten.aurora.gobo.integration.postOrNull
+import org.springframework.stereotype.Service
+import org.springframework.web.reactive.function.client.WebClient
+
+private val logger = KotlinLogging.logger {}
+
+enum class NagHubColor(val hex: String) {
+    Red("#FF0000"),
+    Green("#008000"),
+    Yellow("#FFFF00");
+
+    @JsonValue
+    override fun toString(): String = hex
+}
+
+data class DetailedMessage(
+    val color: NagHubColor,
+    val text: String
+)
+
+data class SendMessageRequestNagHub(
+    val simpleMessage: String? = null,
+    val detailedMessages: List<DetailedMessage>
+)
+
+data class NagHubResult(
+    val success: Boolean
+)
+
+@Service
+class NagHubService(
```

Comment:
denne bør være mulig å disable slik som herkimer, dbh og skap

Reply:
Legge til samme opplegge som for dbh og skap da eller?

Reply:
Jepp RequiresNagHub tenkte jeg

Reply:
Skal få lagt det til

Reply:
lag til